components/ConfirmDelete/index.js

```diff
@@ -23,7 +23,7 @@ export default function ConfirmDelete({ handleDeleteTrip }) {
   }
   return (
     <div>
-      <p>Do you really want to delete this trip?</p>
+      <p>Are you sure?</p>
```

Comment:
Personally, I would keep this question as it was cause the user knows exactly what action he is confirming/aborting. If you are using this component also for other use-cases you could pass this message as a prop.

Reply:
A shorter message fits our design better.